modules/chanserv/clear_akicks.c

```diff
@@ -33,12 +33,6 @@ cs_cmd_clear_akicks(struct sourceinfo *si, int parc, char *parv[])
 		return;
 	}
 
-	if (metadata_find(mc, "private:close:closer"))
-	{
-		command_fail(si, fault_noprivs, STR_CHANNEL_IS_CLOSED, name);
-		return;
-	}
-
 	if (!mc->chan)
 	{
 		command_fail(si, fault_nosuch_target, _("\2%s\2 does not exist."), name);
```

Comment:
One last thing; please replace this with

```C
command_fail(si, fault_nosuch_target, STR_CHANNEL_IS_EMPTY, name);
```

... to match the other checks for network channel existence in this PR.

Reply:
Also fixed along with the two in clone.c.